go-controller/pkg/ovn/secondary_layer3_network_controller.go

```diff
@@ -747,6 +751,30 @@ func (oc *SecondaryLayer3NetworkController) addUpdateRemoteNodeEvent(node *kapi.
 	return err
 }
 
+// addNodeSubnetEgressSNAT adds the SNAT on each node's switch in L3 networks
+// snat eth.dst == 76:23:f 169.254.0.12 10.128.0.0/24
```

Comment:
the ethernet match isnt a real mac address. Is that intentional? Describe the mac is the management port mac?

Reply:
ah no when we do ovn-nbctl lr-nat-list it truncates the full mac :D it only shows me that much :) I can put a full version here so that its not confusing

Reply:
done! gave a more complete mac here